### PR TITLE
docs: add list of all core permissions [issue #2966]

### PR DESCRIPTION
**Pull request by 514sid**
Related to issue https://github.com/tauri-apps/tauri-docs/issues/2966

### CODE REVIEW

**Comment by FabianLars** [comment]
Thanks! I think for the future automating this may make sense (like we do for the plugins) 🤔 But for now this is totally fine.

@tweidinger idk if you care but are you okay with this being on the `security/capabilities/` page or would you prefer some other location?

**Comment by tweidinger** [comment]
I think this kind of reference should be part of a [`reference/acl/core-capabilities.mdx`](https://github.com/tauri-apps/tauri-docs/tree/v2/src/content/docs/reference/acl) file.
Agreed on it is okay to have it manually for now but should be automatically generated from the already generated markdown of the crate (https://github.com/tauri-apps/tauri/blob/dev/crates/tauri/permissions/app/autogenerated/reference.md for example).

This shouldn't live in the guide section where the concept of a capability is explained.

**Comment by 514sid** [comment]
> I think this kind of reference should be part of a [`reference/acl/core-capabilities.mdx`](https://github.com/tauri-apps/tauri-docs/tree/v2/src/content/docs/reference/acl) file.

If the list of capabilities is added to this file, I think it would be nice to include a link to it at `/security/capabilities/`

File: src/content/docs/reference/.gitignore [hunk 1]
```diff
@@ -1,5 +1,4 @@
 config.md
-acl/*.md
 cli.mdx
```
Comment (FabianLars) [inline src/content/docs/reference/.gitignore:2]:
```suggestion
cli.mdx
acl/*.md
!acl/core-permissions.mdx
```

idk if the "un-ignore" line is really needed with the different file extensions but the auto-generated .md files should still be ignored.

Comment (514sid) [reply src/content/docs/reference/.gitignore:2]: I think this change wasn't needed, you're right

File: src/content/docs/security/capabilities.mdx [hunk 2]
```diff
@@ -227,3 +227,7 @@ Everything can be inlined into the `tauri.conf.json` but even a
 little more advanced configuration would bloat this file and
 the goal of this approach is that the permissions are abstracted
 away whenever possible and simple to understand.
+
+## Core Permissions
+
+List of all core permissions can be found on the [Core Permissions](/reference/acl/core-permissions/) page.
```
Comment (FabianLars) [inline src/content/docs/security/capabilities.mdx]:
```suggestion
A list of all core permissions can be found on the [Core Permissions](/reference/acl/core-permissions/) page.
```

**Review by FabianLars — APPROVED** [review]
thanks :)

**Pull request merged**